go.mod

```diff
@@ -0,0 +1,81 @@
+module github.com/filecoin-project/go-filecoin
+
+go 1.12
+
+require (
+	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
+	github.com/Microsoft/go-winio v0.4.12 // indirect
+	github.com/docker/distribution v2.7.1+incompatible // indirect
+	github.com/docker/docker v0.7.3-0.20190315170154-87d593639c77
+	github.com/docker/go-connections v0.4.0 // indirect
+	github.com/docker/go-units v0.3.3 // indirect
+	github.com/filecoin-project/go-leb128 v0.0.0-20190212224330-8d79a5489543
+	github.com/golangci/golangci-lint v1.15.0
+	github.com/gorilla/mux v1.7.0 // indirect
+	github.com/gxed/pubsub v0.0.0-20180201040156-26ebdf44f824
+	github.com/ipfs/go-bitswap v0.0.1
```

Comment:
Are we stuck treating all existing software as a patch?  Is there work in progress to bump up the semantic version of existing packages to either their old gx version, or something more meaningful, e.g. start with major version 1?

Reply:
@ZenGround0 in libp2p and ipfs we made a decision to restart version numbering. In gx versions did not have real meaning and we treated them as such (packages with major version 7 as an example). We've set all versions to 0.0.1 and we will work from there.